notebook.ipynb

```diff
@@ -933,13 +933,563 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Dealing with missing values"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# checking for null values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "ClientID                0.000000\n",
+       "BirthYear               0.000000\n",
+       "AcademicLevel           0.000000\n",
+       "RelationshipStatus      0.000000\n",
+       "AnnualIncome            1.071429\n",
+       "ChildrenAtHome          0.000000\n",
+       "TeensAtHome             0.000000\n",
+       "EnrollmentDate          0.000000\n",
+       "LastPurchaseDays        0.000000\n",
+       "WineSpending            0.000000\n",
+       "FruitSpending           0.000000\n",
+       "MeatSpending            0.000000\n",
+       "FishSpending            0.000000\n",
+       "SweetSpending           0.000000\n",
+       "GoldSpending            0.000000\n",
+       "DiscountedPurchases     0.000000\n",
+       "WebPurchases            0.000000\n",
+       "CatalogPurchases        0.000000\n",
+       "StorePurchases          0.000000\n",
+       "WebVisitsMonth          0.000000\n",
+       "Campaign3Success        0.000000\n",
+       "Campaign4Success        0.000000\n",
+       "Campaign5Success        0.000000\n",
+       "Campaign1Success        0.000000\n",
+       "Campaign2Success        0.000000\n",
+       "RecentComplaint         0.000000\n",
+       "Z_CC                    0.000000\n",
+       "Z_R                     0.000000\n",
+       "LastCampaignResponse    0.000000\n",
+       "dtype: float64"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data.isnull().mean()*100"
```

Comment:
The Output shown does not justify the comment,
```bash
'AnnualIncome' has 24 null values.
```

It may be due to usage `.mean()*100` you may try `.sum()` instead.